nyc_taxis/operations/default.json

```diff
@@ -25,6 +25,47 @@
       "retry-until-success": true,
       "include-in-reporting": false
     },
+    {
+      "name": "create-snapshot-repository",
+      "operation-type": "create-snapshot-repository",
+      "repository": "{{ snapshot_repository_name | default('test-repository') }}",
+      "body": {
+        "type": "s3",
+        "settings": {
+          "bucket": "{{ s3_bucket_name | default('opensearch-snapshot') }}",
+          "region": "{{ s3_bucket_region | default('us-east-1') }}"
```

Comment:
Should we provide defaults here? The bucket needs to be pre-created AFAIK so if it is not and the `s3_bucket_name` parameter is omitted, then the track will fail once it reaches this operation. We should be able to fail faster by making these required parameters.

Same comment for any parameters that require user setup

Reply:
Thanks! This is a very useful idea! I removed the default value of `s3_bucket_name` and `s3_bucket_region`, and updated the documentation in commit https://github.com/opensearch-project/opensearch-benchmark-workloads/pull/58/commits/53178a45dfa7bdcd7ef66831d53bb962c22e7f94.